static/panes/executor.ts

```diff
@@ -215,7 +215,7 @@ export class Executor extends Pane<ExecutorState> {
     }
 
     compilerIsVisible(compiler: CompilerInfo): boolean {
-        return !!compiler.supportsExecute;
+        return true;
```

Comment:
should this not somehow be `compiler.supportsExecute || compiler.supportsBinary` equivalent like it is in `compiler.ts` (or some better named function inside `compiler` itself?